deploy/olm-catalog/ibm-block-csi-operator/1.8.0/metadata/annotations.yaml

```diff
@@ -5,5 +5,5 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: ibm-block-csi-operator
-  com.redhat.openshift.versions: "v4.7-v4.9"
-  com.redhat.delivery.operator.bundle: true
+
+  com.redhat.openshift.versions: v4.7-v4.9
```

Comment:
no need for it in community files?

Reply:
according to @oriyarde the answer is no, no need to implement these changes for community olm as well